auto_install/install.sh

```diff
@@ -95,11 +99,11 @@ export LC_ALL=C
 
 main(){
 	# Pre install checks and configs
+	distroCheck
```

Comment:
Distro check should be done after root check. 
there is no point in checking for distro if there are no permissions to run it.

Reply:
I did it because the `distroCheck` function sets some variables that are used downstream (also in the `rootCheck` function)

Reply:
doesn't make much sense to be, but don't think its any blocker or anything major to worry about. will resolve this point then.